Dockerfile

```diff
@@ -21,8 +21,8 @@ RUN apk upgrade --update && \
   npm install -g node-gyp
 
 # install codebase
-RUN (cd ${LOCALDIR} ; npm install ; node-gyp rebuild)
+WORKDIR "${LOCALDIR}"
+RUN npm install --unsafe-perm
```

Comment:
Moved the `WORKDIR` directive up so I didn't have to do that `cd`, and added the `--unsafe-perm` because building was not allowed as root.  This only matters when installed as root, or in a docker container (again, as root).